src/protocols/two_party_ecdsa/lindell_2017/party_one.rs

```diff
@@ -65,6 +65,7 @@ pub struct PaillierKeyPair {
 pub struct Signature {
     pub s: BigInt,
     pub r: BigInt,
+    pub recid: u8,
```

Comment:
it seems that your `recid` is just a single bit. consider change the type to `bool` ?

Reply:
I want to eventually use it for arithmetic operations... 
Moreover there are not space savings, just safe type checking...